etl.js

```diff
@@ -146,7 +145,39 @@ Etl.upsertChecklistItem = async function _upsertChecklistItem(item) {
     );
     projectMeta.amount = item._amount;
     store.set(`${ISSUE_TO_ITEM}:${item.id}:project`, projectMeta);
-    await sleep(SLEEP);
+  }
+
+  let secondaryAdminID = "5ff85abd2b962872d01fe3bf";
+  let fallbackOwnerField = card.customFieldItems.find(function (card) {
+    return card.idCustomField === secondaryAdminID;
+  });
+  let fallbackOwner = fallbackOwnerField?.value?.text;
+  let isExpectedFallbackOwner = fallbackOwner === projectMeta.fallback_owner;
+  if (fallbackOwner && !isExpectedFallbackOwner) {
+    await gh.projects.setFallbackOwner(
+      projectMeta.projectItemNodeId,
+      fallbackOwner
+    );
+    projectMeta.fallback_owner = fallbackOwner;
+    store.set(`${ISSUE_TO_ITEM}:${item.id}:project`, projectMeta);
+  }
+  let fallbackOwnerId = "";
+  let ownerArray = card.idMembers;
+  if (fallbackOwner) {
+    fallbackOwnerId = Transform.usernameToId(fallbackOwner);
+  }
+
+  let ownerIdArray = ownerArray.filter(function (ownerId) {
```

Comment:
Same comment here as before. Pluralize to indicate array. Don't name things literally Array (except in rare occasion when you're differentiating between a map collection and an array collection).